### ISSUE TITLE
feature(toolkit): hopeit.engine events client

### ISSUE THREAD

**Issue by leosmerling-hopeit** [hopeit-git/hopeit.engine#42]
MOTIVATION:
There is one use case where hopeit.engine apps needs to call another method, form the same or different app.

Since the HTTP request details are mosty known by the engine, the idea is to create a client.py module, that allow developers to make sync HTTP calls to other events or apps using just app and event names.

EXAMPLE:

```
from hopeit.toolkit.events.client import EventClient

def __init_event__(context: EventContext):
   client = EventClient(context=context, app="other_app", event_name="other_event")

def my_event(payload: ..., context: EventContext) -> ...:
   with client.session(context) as session:
      data = session.execute_event(payload)
  ... do something with data ...
  return my_data
```

CONSIDERATIONS:
- It should be based on aiohttp (same as server)
- EventClient should be able to determine the proper endpoint base on config from context where created.
- App configuration could have a client config section detailing host names, etc.
- Client load balancing could be considered
- This client can be used by external apps also

**Comment by leosmerling-hopeit** [comment]
This feature is being developed in #94

**Comment by leosmerling-hopeit** [comment]
Closed by #94
